### PR TITLE
Information icon - In the preset and team information icon add some clarification related to Service Connection

### PR DESCRIPTION
**Pull request by RahulPidde23**
**Changes**
Changes done related to Preset and Team icon information.

**Test Cases**
- Open Pipeline and check Preset Team icon information

### CODE REVIEW

File: CxScan/CxScanV20/task.json [hunk 1]
```diff
@@ -168,7 +168,7 @@
       "type": "string",
       "label": "Team",
       "required": false,
-      "helpMarkDown": "The full team name (for example, CxServer\\SP\\Company\\Users). If team is mentioned in Service connection,then this will get overridden with Service connection Team",
+      "helpMarkDown": "The full team name (for example, CxServer\\SP\\Company\\Users). In Service Connection if Team(optional) value is added , then it will igonres Team available in pipeline and uses Team available in service connection only.If  Team is blank in service connection then it will use pipelines Team.",
```
Comment (ThokalSameer) [inline CxScan/CxScanV20/task.json:171]: Check the spacing between the words.

File: vss-extension.json [hunk 2]
```diff
@@ -128,7 +128,7 @@
               {
                 "id": "preset",
                 "name": "Preset",
-                "description": "predefined sets of queries that you can select when Creating, Configuring and Branching Projects. Predefined presets are provided by Checkmarx and you can configure your own. You can also import and export presets (on the server).",
+                "description": "Predefined sets of queries that you can select when Creating, Configuring and Branching Projects. Predefined presets are provided by Checkmarx and you can configure your own. You can also import and export presets (on the server).In Service Connection if preset(optional) value is added , then it will igonres Preset available in pipeline and uses preset available in service connection only.If  Preset is blank in service connection then it will use pipelines preset.",
```
Comment (ThokalSameer) [inline vss-extension.json:131]: Remove the extra space between "If   Preset" from all the above messages.

File: CxScan/CxScanV20/task.json (hunk 1 above)
Comment (RahulPidde23) [reply CxScan/CxScanV20/task.json]: Resolved

File: vss-extension.json (hunk 2 above)
Comment (RahulPidde23) [reply vss-extension.json]: Resolved

**Review by ThokalSameer — APPROVED** [review]
Approved

**Pull request merged**